protocol/0014-ORDT-order_types.md

```diff
@@ -388,6 +388,9 @@ In Spot market, for multiple iceberg orders submitted as a batch of orders with
 - A Stop order that hasn't been triggered can be cancelled. (<a name="0014-ORDT-071" href="#0014-ORDT-071">0014-ORDT-071</a>)
 - All stop orders for a specific party can be cancelled by a single stop order cancellation. (<a name="0014-ORDT-072" href="#0014-ORDT-072">0014-ORDT-072</a>)
 - All stop orders for a specific party for a specific market can be cancelled by a single stop order cancellation. (<a name="0014-ORDT-073" href="#0014-ORDT-073">0014-ORDT-073</a>)
+- A party with a long position enters a buy stop order with position size override and is valid but at the time of trigger it is cancelled then the stop order should be cancelled. (<a name="0014-ORDT-137" href="#0014-ORDT-137">0014-ORDT-137</a>)
+- If a stop order is placed with a position_fraction equal to 0.5 and the order size is 2.5 then the rounding should be equal to 3 (<a name="0014-ORDT-138" href="#0014-ORDT-138">0014-ORDT-138</a>)
```

Comment:
```suggestion
- If a stop order is placed with a position_fraction equal to 0.5 and the order size is 5 then the rounding should be equal to 3 (<a name="0014-ORDT-138" href="#0014-ORDT-138">0014-ORDT-138</a>)
```

Reply:
@davidsiska-vega @peterbarrow - please can you confirm this change is correct. Looks sensible to me.

Reply:
Looks good to me, although maybe it would be clearer to call it the position size of 5 instead of the order size.